### DISCUSSION TITLE
Craft Guides: Add guide for using live preview across multiple domains

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/docs#292] [Ideas]
### Description
CraftCMS introduced a new featured in 3.1 to enable the ability for live preview to work on multi site setups when previewing entries that reside on other domains (in addition to the CP domain). Other than the linked guide below (specifically mentioning config for multiple sub domains), there is no documentation to explain what needs to be setup to enable this new feature. The guide should be amended to include this information.

https://craftcms.com/guides/using-live-preview-across-multiple-subdomains

**Comment by matt-adigital** [comment]
Also ran into this using Craft 3.3.3 and couldn't find relevant documentation for it. We have 2 sites on completely different domains but powered by the same control panel. I've not made any changes to the cp domain, but I've tweaked the htaccess file on the additional domain (without a cp) to include the following:

```
Header unset X-Frame-Options
Header always set X-Frame-Options "ALLOW-FROM https://www.cpdomain.co.uk"
Header set Content-Security-Policy "frame-ancestors www.cpdomain.co.uk"
```

Obviously replace cpdomain in the code example with your actual cp domain if you're going to use this. This has now allowed live preview to work for our additional sites now. Some browsers such as IE may not support this solution but for now it allows our clients to access the live preview for all their domains. If there is a better way of achieving this for multiple domains then it would be good to see this documented or added as a guide. Thanks.
